pkg/driver/krun/register.go

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package krun
+
+import "github.com/lima-vm/lima/v2/pkg/registry"
+
+func init() {
+	registry.Register(New())
```

Comment:
This is only needed for built-in internal drivers, it is not needed for external drivers.